js/src/d3-bundle.js

```diff
@@ -0,0 +1,42 @@
+import {
+  event,
+  select,
+  selectAll
+} from "d3-selection";
+
+import {
+  xhr,
+  json
+} from 'd3-xhr';
+
+import {
+  range,
+  sum,
+  ascending,
+  descending
+} from 'd3-arrays';
+
+import * as scale from 'd3-scale';
+
+import { format } from 'd3-format';
+
+const functor = function(x) {
```

Comment:
'const' is available in ES6 (use esnext option) or Mozilla JS extensions (use moz).